contracts/optimistic-ethereum/OVM/bridge/messaging/OVM_L1CrossDomainMessenger.sol

```diff
@@ -49,6 +49,7 @@ contract OVM_L1CrossDomainMessenger is iOVM_L1CrossDomainMessenger, Abs_BaseCros
     {
```

Comment:
Funny that we have an `initialize()` function, but no `isInitialized()` checks on the other functions.